pkg/streamingpromql/operators/functions/histograms.go

```diff
@@ -219,7 +218,13 @@ func (h *HistogramFunctionOverInstantVector) accumulateUntilGroupComplete(ctx co
 		// It is also possible that both series groups are the same.
 		// The conflict in points is then detected in computeOutputSeriesForGroup.
 		h.saveNativeHistogramsToGroup(s.Histograms, thisSeriesGroups.nativeHistogramGroup)
-		h.saveFloatsToGroup(s.Floats, thisSeriesGroups.bucketValue, thisSeriesGroups.classicHistogramGroup)
+		if thisSeriesGroups.classicHistogramGroup != nil {
```

Comment:
If there is no `le` label, but there are float values for the series, do we need to emit an annotation or do something like that here?

Reply:
Yes. This got complicated without tracking both groups regardless of labels in seriesGroupPairs, so I've added that back in 29e883e65b47a5079062c11ab34b70e3252648a8